pkgs/development/python-modules/mpi4py/default.nix

```diff
@@ -34,9 +34,10 @@ buildPythonPackage rec {
     mpi
   ];
 
-  pythonImportsCheck = [ "mpi4py" ];
+  # Some tests are known to fail when using MPICH
```

Comment:
Could you add a little bit more information about this please? Are there any failures that are known by upstream and you can link to? Do they wish to support other MPI implementations? If not, and they simply state that in their docs that only openmpi is supported / tested, then a link to this statement would be nice, along with a small quote of the failures you encountered.